packages/faustwp-core/tests/hooks/useAuth.test.ts

```diff
@@ -47,6 +47,21 @@ describe('useAuth hook', () => {
       status: 200,
       body: JSON.stringify({ accessToken: 'at', refreshToken: 'rt' }),
     });
+    fetchMock.get(`http://headless.local/index.php?graphql&query=query%20Viewer%20%7B%0A%20%20viewer%20%7B%0A%20%20%20%20name%0A%20%20%20%20username%0A%20%20%20%20capabilities%0A%20%20%20%20databaseId%0A%20%20%20%20description%0A%20%20%20%20email%0A%20%20%20%20firstName%0A%20%20%20%20id%0A%20%20%20%20lastName%0A%20%20%20%20nickname%0A%20%20%20%20locale%0A%20%20%20%20registeredDate%0A%20%20%20%20slug%0A%20%20%20%20templates%0A%20%20%20%20uri%0A%20%20%20%20url%0A%20%20%20%20userId%0A%20%20%20%20__typename%0A%20%20%7D%0A%7D&operationName=Viewer&variables=%7B%7D`, {
```

Comment:
It may be challenging to programmatically recreate this URL in tests. You would have to encode the URL params for the Apollo `query` and `variables`. Additionally, Apollo strips whitespace from their queries before creating the urls, so even if we did something like:

```
const urlSearchParams = new URLSearchParams({
  query: encodeURIComponent(print(VIEWER_QUERY)),
  variables: encodeURIComponent(JSON.stringify({})),
});
```

it still wouldn't match the URL since the whitespace would differ from the Apollo generated URL.